### ISSUE TITLE
Add colors to the style guide palette

### ISSUE THREAD

**Issue by karlaturcios** [18F/identity-style-guide#118]
On the data viz side, it would be awesome to have:
1. A color for Column Charts, Line Graph
2. 4-6 colors for Pie Chart, donut chart
3. A neutral gray/black for the axis and the labels
4. data tile (shows a statistic in large font size, with smaller sized label and description)
For data tables:
1. color for data values
2. a color data labels
Also would be great to consider colors for pills:
1. pill with a link 
2. success pill
3. warning Pill
4. error pill
5. info pill
*note on pills, common application would be on white background with white text inside colored pill

Additional items:

- Required field item *

**Comment by nickttng** [comment]
Tracking this resource here (instead of losing it in Slack): [Data Visualization Standards (beta) - Colors](https://xdgov.github.io/data-design-standards/components/colors)

Consider creating login data viz palette by seeing how DVS breaks its palette down to:

- Featured colors
- Sequential colors
- Qualitative colors
- Diverging colors

**Comment by karlaturcios** [comment]
To illustrate the constraints that come up with using current colors for data views:
<img width="961" alt="Screen Shot 2019-11-07 at 11 50 57 AM" src="https://user-images.githubusercontent.com/1444176/68409341-e808e880-0154-11ea-8744-0843f1590516.png">

**Comment by nickttng** [comment]
@karlaturcios Now that we have additional color palettes via #121. Should we close this and open any new issues for additional needs?

**Comment by nickttng** [comment]
Closing this for now. We can reopen or create a new issue if new color issues arise
